{"uuid": "d00be3de-e3a1-430b-9488-030b0de0ed15", "children": ["c201efeb-d26e-4cd2-bbdd-db46533dc8e2"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: 'geckodriver' executable needs to be in PATH. \n\n", "trace": "  File \"/Users/pramod/PycharmProjects/PyWebAutomation0x/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/pramod/PycharmProjects/PyWebAutomation0x/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/pramod/PycharmProjects/PyWebAutomation0x/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/pramod/PycharmProjects/PyWebAutomation0x/tests/vwoLoginTests/conftest.py\", line 8, in setup\n    driver = webdriver.Firefox()\n             ^^^^^^^^^^^^^^^^^^^\n  File \"/Users/pramod/PycharmProjects/PyWebAutomation0x/venv/lib/python3.11/site-packages/selenium/webdriver/firefox/webdriver.py\", line 164, in __init__\n    self.service.start()\n  File \"/Users/pramod/PycharmProjects/PyWebAutomation0x/venv/lib/python3.11/site-packages/selenium/webdriver/common/service.py\", line 81, in start\n    raise WebDriverException(\n"}, "start": 1693221647564, "stop": 1693221647575}], "start": 1693221647563, "stop": 1693221647711}